utilities/runvx/README.md

```diff
@@ -33,6 +33,7 @@ If available, this project uses OpenCV for camera capture and image display.
       -frames:[<start>:]<end>|eof|live
           Run the graph/node for specified frames or until eof or just as live.
           Use live to indicate that input is live until aborted by user.
+          Use -frames:[amount of frames]:ignore-eof when you want to run on the same image multiple times.
```

Comment:
I would change to [number of frames] instead

Reply:
I updated the PR. I changed to [number of frames] and I added the instructions for ignore-eof in runvx.cpp.
I will check the runvx code to see if more than just the ignore-eof command line argument needs to be changed to "loop"

Reply:
Changed ignore-eof to loop and put that in both the README.md and the runvx help.